usecases/service_accounts_test.go

```diff
@@ -63,297 +64,329 @@ type saHasPermissionTestCase struct {
 var saHasPermissionTestCases = []saHasPermissionTestCase{
 	// No permissions
 	saHasPermissionTestCase{
+		name:     "No Permissions",
 		regPs:    nil,
 		test:     "Service1::RL::Do1::x::*",
 		expected: false,
 	},
 	// != Service
 	saHasPermissionTestCase{
+		name:     "Different Service",
 		regPs:    []string{"Service1::RL::Do1::x::*"},
 		test:     "Service2::RL::Do1::x::*",
 		expected: false,
 	},
 	// Toying around with actions, * and multiple layers in RH
 	saHasPermissionTestCase{
+		name:     "Wildcard operator Scenario 1",
```

Comment:
Could we be more descriptive here? What's "scenario 1"? Try going for scenario_outcome, eg, "resource_wildcard_match" , "resource_wildcard_noMatch", etc.

Ditto for all following names.

Reply:
Completely agree 👍 . I put these names as I was not sure about what each scenario was testing in these tests.